src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationTestsExtension.java

```diff
@@ -20,7 +20,7 @@
 @Slf4j
 public class IntegrationTestsExtension implements BeforeAllCallback, BeforeEachCallback {
 
-    private final String DLQ_PREFIX = "DLQ.";
+    private final String dqlPrefix = "DLQ.";
```

Comment:
Could we make this `static` and retain the original uppercase name?

Reply:
Implemented in b0527079

Reply:
Also `DLQ` (Dead Letter Queue), not `DQL` (data query language?) :)

Reply:
Good spot, have fixed this in f0c529e